JS.md

```diff
@@ -394,6 +402,8 @@ aliases `M.[[Memory]]` and whose
 [[[ArrayBufferByteLength]]](http://tc39.github.io/ecma262/#sec-properties-of-the-arraybuffer-prototype-object)
 is set to the new byte length of `M.[[Memory]]`.
 
+Return `ret`.
```

Comment:
I think you mean `ToJSValue` (because we're starting with a wasm `i32`), but point taken, I'll add this.